src/repo.rs

```diff
@@ -77,13 +79,33 @@ pub fn repo_sync(repo: &str, force: bool) -> Result<()> {
 
     loop {
         println!("Running repo sync...");
-        let cmd = Command::new("repo")
+        let repo_sync = format!("repo sync -j{}", &num_cpus::get());
+        // `script` is a Unix command that takes a copy of all output to the terminal
+        // and writes it to `typescript` file.
+        // Below, explanation of `script` options.
+        // -q  Be quiet (do not write start and done messages to standard output).
+        // -e Return the exit status of the child process.
+        // -f Flush output after each write.
+        // -c Run the command rather than an interactive shell. This makes it easy for a
+        // script to capture the output of a program that behaves differently when its
+        // stdout is not a tty.
+        let mut cmd = Command::new("script")
             .current_dir(repo)
             .stdout(Stdio::piped())
             .stderr(Stdio::piped())
-            .args(["sync", "-j", &num_cpus::get().to_string()])
+            .args(["-qefc", &repo_sync])
             .spawn()
             .context("Failed to execute repo sync")?;
+
+        let stdout = cmd.stdout.take().unwrap();
+        let reader = BufReader::new(stdout);
+        let split_iter = reader
+            .split(b'\r')
```

Comment:
is it possible to split on either \r or \n?

That way we split on newlines as well and print more readily before the first \r

Reply:
It isn't possible to separate by one of multiple byte characters using `split()`, so I reverted how to output progress. 
a981ea1080e4110bb3eee1dc55a96768c91840b0

Reference: 
https://doc.rust-lang.org/std/io/trait.BufRead.html#method.split

Reply:
(If we add a progress bar, it may be easier to implement it by splitting the output with `\r`...)